vic/drivers/classic/src/alloc_atmos.c

```diff
@@ -37,73 +37,73 @@ alloc_atmos(int                 nrecs,
 {
     int i;
 
-    *atmos = (atmos_data_struct *) calloc(nrecs, sizeof(atmos_data_struct));
+    *atmos = calloc(nrecs, sizeof(atmos_data_struct));
     if (*atmos == NULL) {
         log_err("Memory allocation error in alloc_atmos().");
     }
 
     for (i = 0; i < nrecs; i++) {
-        (*atmos)[i].air_temp = (double *) calloc(NR + 1, sizeof(double));
+        (*atmos)[i].air_temp = calloc(NR + 1, sizeof((*atmos)[i].air_temp));
```

Comment:
I don't think this is right - the left size and the right size have the same element size. There is a `*` missing on the right side. Note that I don't think that travis or the compiler will necessarily catch this, since I am free to allocate any size of memory I want. Please see the earlier discussion that we had as part of this PR.